composer.json

```diff
@@ -86,13 +89,6 @@
         "drupal-scaffold": {
             "locations": {
                 "web-root": "web/"
-            },
-            "file-mapping": {
-                "[web-root]/sites/default/settings.php": {
-                    "mode": "append",
-                    "default": "web/core/assets/scaffold/files/default.settings.php",
-                    "append": "assets/patches/default_settings.txt"
```

Comment:
Should probably delete the related file ( https://github.com/Islandora-Devops/islandora-starter-site/blob/main/assets/patches/default_settings.txt ), if it's no longer in use; though, on the other hand, could be leading to the failure to detect the configuration, as it is what is presently expected to inform Drupal of where its configuration is.